tracee/tracee.go

```diff
@@ -473,22 +473,33 @@ func (t *Tracee) processEvents() {
 				t.handleError(err)
 				continue
 			}
+			argsNames := make([]string, ctx.Argnum)
+			argsTags := make([]uint8, ctx.Argnum)
 			args := make([]interface{}, ctx.Argnum)
 			for i := 0; i < int(ctx.Argnum); i++ {
-				args[i], err = readArgFromBuff(dataBuff)
+				argTag, argVal, err := readArgFromBuff(dataBuff)
 				if err != nil {
 					t.handleError(err)
 					continue
 				}
+				argsTags[i] = argTag
+				args[i] = argVal
+				argName, ok := argNames[argTag]
+				if ok {
+				       argsNames[i] = argName
```

Comment:
The event processing code doesn't care about the arg names, it just needs their tags. the names are for printing only, so I think this code doesn't belongs here rather in the `newEvent` code

Reply:
You are right. I'm handling exactly this in my next PR